jupyter-notebooks/basemaps_contributing_scene_metadata.ipynb

```diff
@@ -0,0 +1,478 @@
+    "\n",
+    "Basemap quads and their contributing scene information can also be viewed in the [Planet Basemaps Viewer](https://www.planet.com/basemaps/) web application, although this script makes it easier to get data for many quads and larger regions."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "92ccb198",
+   "metadata": {},
+   "source": [
+    "### Import Pacakges"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4a351534",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import packages\n",
+    "import os\n",
+    "import json\n",
+    "import requests\n",
+    "from requests.auth import HTTPBasicAuth"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f621ce5",
+   "metadata": {},
+   "source": [
+    "### Setup Authentication to Planet APIs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fd44e418",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can use an environment variable saved on our computer to store our API key\n",
+    "PLANET_API_KEY = os.getenv('PL_API_KEY')\n",
+    "\n",
+    "# Alternatively, we can just paste in our API \n",
+    "# PLANET_API_KEY = \"YOUR_API_KEY_HERE\"\n",
+    "\n",
+    "# We can use our API key as the \"username\" in the basic auth method for our requests\n",
+    "auth = HTTPBasicAuth(PLANET_API_KEY, '')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cfa9a470",
+   "metadata": {},
+   "source": [
+    "### Use the Basemaps API to get quads"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "67021bd8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Setup a base url for the Planet Basemaps API\n",
+    "BASE_URL = 'https://api.planet.com/basemaps/v1/mosaics/'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e639e329",
+   "metadata": {},
+   "source": [
+    "Next we can make a request to get our basemap. We'll need to either know or retrieve the basemap's Id. In the cell below, we can find the basemap's Id by using the `name__is` query parameter with the list [`/mosaics`](https://developers.planet.com/docs/basemaps/reference/#operation/listMosaics) endpoint on the Basemaps API. If you already know the Id of the basemap, you can skip this part and set the `BASEMAP_ID` in subsequent cells.\n",
+    "\n",
+    "*Note: \"basemaps\" and \"mosaics\" can be referred to interchangeably* "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8c5d6c1b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get basemap by name\n",
+    "BASEMAP_NAME = 'global_monthly_2022_05_mosaic'\n",
```

Comment:
Is this basemap one that's available to the public, or one that people would need access for?